client/src/component/Footer.css

```diff
@@ -0,0 +1,102 @@
+  .footer-section {
+    padding: 23px;
+    text-align: center;
+    position: relative;
+    bottom: 0;
+    width: 100%;
+    height: 100%;
+    background-color: var(--white);
+    display: block;
+    font-size: 1.2em;
+    box-shadow: var(--black);
+    z-index: 800;
+  }
+
+  .Footer {
+    background: var(--cyf-color);
+    color: var(--white);
+    height: auto;
+    font-size: 1em;
+    font-weight: 600;
+    line-height: 1.5em;
+    position: relative;
+    text-transform: none;
+    margin: 0 auto;
+    padding: 8px;
+    border: 0;
+    border-radius: 2px;
+    width: auto;
+    cursor: pointer;
+    font-family: inherit;
+    letter-spacing: 0;
+    white-space: nowrap;
+    transition: padding 0.25s;
+    -webkit-transition: padding 0.25s;
+    text-shadow: none;
+    box-shadow: none;
+  }
+
+  .github,
+  .email {
+    color: var(--gray);
+    padding: 2px;
+    display: flex;
+    justify-content: space-between;
+    margin-right: 4px;
+    position: relative;
+    top: 5px;
+  }
+
+  .fb :hover,
+  .tw :hover,
+  .link :hover,
+  .instag :hover,
+  .github :hover,
+  .email :hover {
+    color: var(--cyf-color);
+  }
+
+  .email-required {
+    background: var(--white);
+    height: auto;
+    font-size: 1em;
+    line-height: 1.5em;
+    position: relative;
+    text-transform: none;
+    margin: 0 auto;
+    padding: 5px;
+    border: 0;
+    border-radius: 2px;
+    width: 300px;
+    cursor: pointer;
+    font-family: inherit;
+    letter-spacing: 0;
+    white-space: nowrap;
+    transition: padding 0.25s;
+    -webkit-transition: padding 0.25s;
+    text-shadow: none;
+    box-shadow: none;
+  }
+
+  .newsletter {
+    margin: 10px;
+  }
+
+  .footer-bottom footer div div p a:nth-child(1) {
+    color: var(--cyf-color);
+    text-decoration: underline;
+    text-decoration-color: var(--black);
+  }
+
+  .footer-bottom footer div div p a:nth-child(2) {
+    color: var(--cyf-color);
+    text-decoration: underline;
+    text-decoration-color: var(--black);
+  }
+
+  .footer-bottom footer div div p a:nth-child(3) {
+    color: var(--cyf-color);
+    text-decoration: underline;
+    text-decoration-color: var();
+    cursor: pointer;
+  }
```

Comment:
Did you need to create three separate definitions here. They are all pretty much the same other than this one that is missing  the argument in var which will not work